javascript/javascript2/week1/currencyConverter.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Currency Converter Form</title>
+    <link rel="stylesheet" href="currencyConverter.css" />
+  </head>
+  <body>
+    <div class="startPage">
+      <h1>Currency Converter App</h1>
+      <div>
+        <section>
+          <h2>Add New Currency Rate</h2>
+
+          <form id="addNewRateform">
+            <label for="base-currency">Base Currency:</label>
+            <input
+              type="text"
+              id="base-currency"
+              placeholder="e.g., USD"
+              required
+            />
+            <label for="target-currency">Target Currency:</label>
+            <input
+              type="text"
+              id="target-currency"
+              placeholder="e.g., EUR"
+              required
+            />
+            <label for="exchange-rate">Exchange Rate:</label>
+            <input
+              type="number"
```

Comment:
Using a `number` input to enter the currency rate is a very good idea. By default, it changes the values by whole numbers when clicking the up/down arrows. One could argue it's too high for a currency rate. So you can use the [step](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step) attribute to allow numbers with decimals.

<img width="509" alt="image" src="https://github.com/Rajesh-Bhatt/hyf-homework/assets/5467758/94443358-283d-47aa-a056-85cc7e3f8f5a">

Reply:
I used step attribute now. Thank you!